manifests/init.pp

```diff
@@ -0,0 +1,166 @@
+  Optional[Array[String[1]]]  $port_path_whitelist               = undef,
+  Optional[Array[String[1]]]  $port_whitelist                    = undef,
+  Optional[Array[String[1]]]  $pwdless_accounts                  = undef,
+  Optional[String[1]]         $readlink_cmd                      = undef,
+  Integer[0,1]                $rotate_mirrors                    = 1,
+  Optional[Array[String[1]]]  $rtkt_dir_whitelist                = undef,
+  Optional[Array[String[1]]]  $rtkt_file_whitelist               = undef,
+  Enum['THOROUGH','LAZY']     $scan_mode_dev                     = 'THOROUGH',
+  Optional[String[1]]         $scriptdir                         = undef,
+  Optional[Array[String[1]]]  $scriptwhitelist                   = undef,
+  Optional[Array[String[1]]]  $shared_lib_whitelist              = undef,
+  Integer[0,1]                $skip_inode_check                  = 0,
+  Optional[String[1]]         $ssh_config_dir                    = undef,
+  Optional[Array[String[1]]]  $startup_paths                     = undef, #suggestion: /etc/rc.d /etc/rc.local ??
+  Optional[String[1]]         $stat_cmd                          = undef,
+  Optional[Array[String[1]]]  $suspscan_dirs                     = undef,
+  Integer                     $suspscan_maxsize                  = 1024000,
+  String[1]                   $suspscan_temp                     = '/dev/shm',
+  Integer                     $suspscan_thresh                   = 200,
+  Optional[Array[String[1]]]  $suspscan_whitelist                = undef,
+  Optional[Array[String[1]]]  $syslog_config_file                = undef,
+  Optional[String[1]]         $tmpdir                            = undef,
+  Optional[Array[String[1]]]  $uid0_accounts                     = undef,
+  Optional[Array[String[1]]]  $update_lang                       = undef,
+  Integer[0,1]                $update_mirrors                    = 1,
+  Integer[0,1]                $updt_on_os_change                 = 0,
+  Integer[0,1]                $use_locking                       = 1,
+  Optional[Array[String[1]]]  $user_fileprop_files_dirs          = undef,
+  Integer[0,1]                $use_sunsum                        = 0,
+  String[1]                   $use_syslog                        = 'NONE', #recommend authpriv.notice??
+  Integer[0,1]                $warn_on_os_change                 = 1,
+  Optional[String[1]]         $web_cmd                           = undef, #shuold keep? rkhunter auto selects one
+  Integer[0,1]                $whitelisted_is_white              = 0,
+  Optional[Array[String[1]]]  $writewhitelist                    = undef,
+  Optional[Array[String[1]]]  $xinetd_allowed_svc                = undef,
+  Optional[String[1]]         $xinetd_conf_path                  = undef
+) {
+
+  package { 'unhide':
+    ensure => $package_ensure
+  }
+
+  package { 'rkhunter':
+    ensure  => $package_ensure,
+    require => Package['unhide']
+  }
+
+  file { $rkhunter_conf_file:
```

Comment:
Need to ensure that this uses [validate_cmd](https://puppet.com/docs/puppet/5.4/types/file.html#file-attribute-validate_cmd) and runs `rkhunter -C --configfile %`

Reply:
thanks! Didn't know about this one